diff --git a/scripts/gen_build_prop.py b/scripts/gen_build_prop.py
index cf8a856..69698bf 100644
--- a/scripts/gen_build_prop.py
+++ b/scripts/gen_build_prop.py
@@ -126,11 +126,6 @@ def parse_args():
   if config["BuildNumber"].startswith("eng."):
     config["BuildNumber"] = config["DateUtc"]
 
-  config["PihooksGmsFp"] = ""
-  config["PihooksGmsModel"] = ""
-  config["PihooksBuildFp"] = ""
-  config["PihooksBuildModel"] = ""
-
   override_config(config)
 
   append_additional_system_props(args)
@@ -248,11 +243,6 @@ def generate_build_info(args):
   print(f"ro.build.flavor={config['BuildFlavor']}")
 
   print(f"ro.infinity.device={config['InfinityDevice']}")
-  
-  print(f"persist.sys.pihooks_FINGERPRINT={config['PihooksGmsFp']}")
-  print(f"persist.sys.pihooks_MODEL={config['PihooksGmsModel']}")
-  print(f"persist.sys.pihooks_mainline_FINGERPRINT={config['PihooksBuildFp']}")
-  print(f"persist.sys.pihooks_mainline_MODEL={config['PihooksBuildModel']}")
 
   # These values are deprecated, use "ro.product.cpu.abilist"
   # instead (see below).
